docs/api/session.md

```diff
@@ -1,5 +1,7 @@
 # session
 
+> Manage browser cookies, cache, and other session data.
```

Comment:
`session` also includes creating and managing Session itself, instead of just session data. Maybe just "Manage browser sessions".